_bazaar/rse-in-the-digital-humanties-domain.md

```diff
@@ -4,10 +4,11 @@ author:
   - icon: fas fa-fw fa-envelope-square
     label: Email
     url: mailto:rse@fthiery.de
-  name: Florian Thiery
+  name: Florian Thiery / Research Squirrel Engineers
+contributors: We are looking for people who are working as RSE (also if you are not officially named as RSE) in the (digital) humanities / archaeological domain. Contact me for some feedback and if you are interested to join!
 looking: true
-title: RSE in the (Digital) Humanties Domain
+title: RSEs in the Digital Humanties/archaeological Domain
 type: TBC
 ...
 
-There are a lot of RSE working in the Digital Humanities (DH) domian. This slot should help to show other what are you doing as a RSE in the DH.
\ No newline at end of file
+There are a lot of RSEs working in the Digital Humanities (DH) or archaeological domian. Most of them a are not not named as Research Software Engineers even though the develop software for researchers or develop software for their own research. This slot should help to show others what RSEs in DH are doing and hopefully find some new collaborations. There are also some non-institutional groups in the DH which do RSE things just for fun in their free time. One of the groups are the [Research Squirrel Engineers](http://squirrel.link). If this slot will help to increase the ammount of squirrels, this would be awesome!
```

Comment:
```suggestion
There are a lot of RSEs working in the Digital Humanities (DH) or archaeological domian. Most of them a are not not named as Research Software Engineers even though they develop software for researchers or develop software for their own research. This slot should help to show others what RSEs in DH are doing and hopefully find some new collaborations. There are also some non-institutional groups in the DH which do RSE things just for fun in their free time. One of the groups are the [Research Squirrel Engineers](http://squirrel.link). If this slot will help to increase the ammount of squirrels, this would be awesome!
```
it should be `they develop` instead of `the develop`

Reply:
passt, danke!